src/Scrollbar.js

```diff
@@ -315,7 +315,7 @@ class Scrollbar extends React.PureComponent {
   _onMouseDown = (/*object*/ event) => {
     var nextState;
 
-    if (event.target !== ReactDOM.findDOMNode(this.refs.face)) {
+    if (event.target !== ReactDOM.findDOMNode(this._faceRef)) {
```

Comment:
You can remove the ReactDOM.findDOMNode call here.  With the new ref style _faceRef is the DOM node.

Reply:
Sure thing.